contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -2,23 +2,29 @@
 pragma solidity 0.8.19;
 
 import {BaseTest} from "./BaseTest.t.sol";
+import {AutomationRegistryBase2_3 as AutoBase} from "../v2_3/AutomationRegistryBase2_3.sol";
 import {IAutomationRegistryMaster2_3, AutomationRegistryBase2_3} from "../interfaces/v2_3/IAutomationRegistryMaster2_3.sol";
 import {ChainModuleBase} from "../../chains/ChainModuleBase.sol";
+import {IAutomationV21PlusCommon} from "../../interfaces/IAutomationV21PlusCommon.sol";
 
 // forge test --match-path src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol
 
 contract SetUp is BaseTest {
   address[] internal s_registrars;
 
   IAutomationRegistryMaster2_3 internal registry;
+  uint256[] internal upkeepIds;
+  uint256[] internal gasLimits;
+  bytes[] internal performDatas;
+  uint256[] internal balances;
```

Comment:
nitpick, but it looks like these are only used within the `NOPsSettlement` test - I think we can move them to that contract instead of making them global across all tests?

Reply:
In fact, it looks like they're only used in one test yeah? Can we just move them to that one test?